doc/contributing/submitting-changes.chapter.md

```diff
@@ -234,6 +234,15 @@ digraph {
 
 The `master` branch is the main development branch. It should only see non-breaking commits that do not cause mass rebuilds.
 
+The guideline for proposed change is good for `master` branch:
+
+- Rebuild count is less than 1000 dependent 1000.
```

Comment:
I don't understand that sentence.

Reply:
Oh wow. That sentence did not survive the edits /o\. Reworded the whole section and transformed into `Important update with rebuild count less than 1000 dependent packages and does not contain heavyweight packages like chromium or firefox. `